tests/fixtures.py

```diff
@@ -0,0 +1,33 @@
+#
+#   Copyright 2013-2023 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Manager test harness test case fixtures for My Asset Manager.
+"""
+from openassetio.constants import kField_EntityReferencesMatchPrefix
+
+
+KIDENTIFIER = "myorg.manager.my_asset_manager"
+
+fixtures = {
```

Comment:
Get rid of the K if upper cased.

Reply:
I added fixtures for `Test_resolve`

The ManagementPolicy compliance tests don't require any fixtures as far as I can tell. They're running without them (I can tell because changing the management policy implementation to be incorrect causes a failure)

Reply:
Part of the main extension work : https://github.com/elliotcmorris/openassetio-python-manager-template/commit/963920e9af7f7735c78d6aab17fb7745003f003c